bundles/specmate-emfrest/src/com/specmate/emfrest/authentication/IResponseAlteringService.java

```diff
@@ -0,0 +1,10 @@
+package com.specmate.emfrest.authentication;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.core.Response;
+
+import com.specmate.rest.RestResult;
+
+public interface IResponseAlteringService {

```

Comment:
Is this interface needed? There is only one implementing class at the moment. Will there be further implementers?

Reply:
Yes, we need this later for the OAuth-Login.

Reply:
Yes, we'll probably need this for the OAuth-Login. And we might need this in the future for services that need to provide a `Response` rather than a `Result`. The `Response` carries more information like cookies.